handlers/main.yml

```diff
@@ -1,3 +1,4 @@
 - name: restart sshd
   service: name={{ sshd_service_name }} state=restarted
   when: "(ssh_server_enabled|bool)"
+  become: yes
```

Comment:
Adding `become: yes` to the playbook fixes this, so I'd rather not have this here, as mentioned here: https://github.com/dev-sec/ansible-ssh-hardening/pull/81

Reply:
Man, this is such a sucky bug. I don't want to run my roles as root, so I don't use `become: yes` at the playbook level. It's supposed to be fixed, so that you can do an `import_role` and put the `become: yes` on that, but it doesn't work correctly. I think it would work, if there were no `include_role` calls between the import and the calling of the handler, but my implementation uses an `include`. That's why I put the `become` on the handler itself. I take it off, but it means I'm still going to have to fork to get it working for me. Not a huge deal, but not ideal.

Reply:
You're right. Let's keep it in for now and see if someone's got a problem with it.